tests/integration/conftest.py

```diff
@@ -961,11 +1035,26 @@ def workaround_remove_uek():
     Reference issue https://issues.redhat.com/browse/RHELC-1544
     """
     if SystemInformationRelease.distribution == "oracle":
-        subprocess.run(["yum", "remove", "-y", "kernel-uek"], check=False)
+        subprocess.run(
+            ["yum", "remove", "-y", "kernel-uek"], check=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE
+        )
```

Comment:
subprocess is nice, but is there a reason why we don't use `shell()` here? I mean there is inconsistency in that

Reply:
Again, having a comment here would be nice, so I'll include that.
The reasoning behind not using shell was that the fixture has a different scope than the shell fixture, which was causing issues.
Now that the context manager is refactored, it should work with calling it directly, without importing as a fixture.
From my perspective this again is out of scope of this PR, let's experiment with that in a different PR.